app/views/service_providers/_form.html.erb

```diff
@@ -13,11 +13,12 @@
                  label: 'Description',
                  hint: 'A description of the app (may be helpful for your colleagues).' %>
 
-  <%= form.association :group,
+  <%= form.association :team,
```

Comment:
this helper was really tricky.... it builds a form key named `service_provider[group_id]` and I couldn't figure out how to override it to be something like `service_provider[team_id]` (I think it uses a lot of reflection to build that), so I had to have the controllers use the old `group_id` attribute